nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -511,7 +511,7 @@ public void onTrigger(final ProcessContext context, final ProcessSessionFactory
                         whereClause = maxValueClauses.isEmpty() ? "1=1" : StringUtils.join(maxValueClauses, " AND ");
                         Long offset = partitionSize == 0 ? null : i * partitionSize + (useColumnValsForPaging ? minValueForPartitioning : 0);
                         // Don't use an ORDER BY clause if there's only one partition
-                        final String orderByClause = partitionSize == 0 ? null : (maxColumnNames.isEmpty() ? customOrderByColumn : maxColumnNames);
+                        final String orderByClause = partitionSize == 0 ? null : (StringUtils.isNotEmpty(customOrderByColumn) ? customOrderByColumn : maxColumnNames);
```

Comment:
This changes the documented behavior slightly, the `Custom ORDER BY Column` description says it will be ignored if `Max Value Column Names` is set. This is no longer true, it is only ignored now if partition size is zero. Can you update the documentation for the `Custom ORDER BY Column` property to reflect this behavior?

Reply:
Also please update the Max Value Column property description to indicate that it will be ignored if Custom ORDER BY Column is set

Reply:
Thank you for reminding me. I will change the description of `Custom ORDER BY Column`. And I think we do not need to change `Max Value Column` property description, it's a common property used by different processors, and when we use  `Custom ORDER BY Column`, we see the description and we will know what will happen.

Reply:
It's not guaranteed the user will see the Custom ORDER BY doc, if you need to change the description slightly you can start with the existing PropertyDescriptor in the Builder by using `.fromPropertyDescriptor()`, there are some examples in the existing codebase